homeassistant/components/media_player/songpal.py

```diff
@@ -0,0 +1,242 @@
+"""
+Support for Songpal-enabled (Sony) media devices.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.songpal/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.media_player import (
+    PLATFORM_SCHEMA, SUPPORT_SELECT_SOURCE, SUPPORT_TURN_OFF,
+    SUPPORT_VOLUME_MUTE, SUPPORT_VOLUME_STEP, SUPPORT_VOLUME_SET,
+    SUPPORT_TURN_ON, MediaPlayerDevice, DOMAIN)
+from homeassistant.const import (
+    CONF_NAME, STATE_ON, STATE_OFF, ATTR_ENTITY_ID)
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['python-songpal==0.0.6']
+
+SUPPORT_SONGPAL = SUPPORT_VOLUME_SET | SUPPORT_VOLUME_STEP | \
+                  SUPPORT_VOLUME_MUTE | SUPPORT_SELECT_SOURCE | \
+                  SUPPORT_TURN_ON | SUPPORT_TURN_OFF
+
+_LOGGER = logging.getLogger(__name__)
+
+
+SET_SOUND_SETTING = "songpal_set_sound_setting"
+PARAM_NAME = "name"
+PARAM_VALUE = "value"
+
+CONF_ENDPOINT = "endpoint"
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_NAME, default=None): cv.string,
```

Comment:
Remove `default=None`. It's unnecessary for Optional to work and it also won't work since we upgraded voluptuous. The upgraded voluptuous will validate all values passed via `default` with the validator. `None` is not valid for `cv.string`

Reply:
Good to know, thanks!